projects/cdk/package.json

```diff
@@ -36,8 +36,8 @@
         "rxjs": ">=6.0.0"
     },
     "optionalDependencies": {
-        "@angular-devkit/core": ">=15.0.0",
-        "@angular-devkit/schematics": ">=15.0.0",
+        "@angular-devkit/core": "15.2.10",
+        "@angular-devkit/schematics": "15.2.10",
```

Comment:
`devDependencies` - ok

`optionalDependencies` - nope

Reply:
optionalDependencies should be ignored when released new versions?

Reply:
Renovate bot should treat `optionalDependencies` in the same way as `peerDepedencies`.

Could we configure renovate bot in this way ?

Reply:
fixed